app/models/student.rb

```diff
@@ -27,9 +27,10 @@ class Student < ApplicationRecord
   belongs_to :school, class_name: "School", foreign_key: "school_id"
 
   validates :registration_no, :roll_no, :gender, :dob, :guardian_name,
-            :guardian_mobile_no, presence: true
+            :guardian_mobile_no, :message_language, presence: true
```

Comment:
Instead of `message_language`, lets use `preferred_language` field_name.

Reply:
Okay